### DISCUSSION TITLE
Welcome to litehtml Discussions!

### DISCUSSION THREAD

**Discussion by tordex** [litehtml/litehtml#248] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by cirospaciari** [comment]
Hello there I'm Ciro Spaciari author of https://github.com/cirospaciari/socketify.py and I will plan to use litehtml to create a library for Python with HTML/CSS based interfaces.

**Reply by tordex** [reply]
Great! I've used litehtml to create UI for one of my pet projects. This is something like small home server on Raspberry Pi with LCD screen. UI part was created with C++, the _backend_ with python.
